### PR TITLE
virsh_domjobinfo: test --keep-completed option

### PR DESCRIPTION
**Pull request by yanqzhan**
Test --keep-completed option for completed domjobinfo, take live dump
action as a demo.

Signed-off-by: Yanqiu Zhang <yanqzhan@redhat.com>

### CODE REVIEW

**Comment by yanqzhan** [comment]
CASE ID: RHEL-185827 - Query completed domain job info with --keep-completed option
description:   --keep-completed  don't destroy statistics of a recently completed job when reading, usage: [--completed [--keep-completed]]
1. use --completed  --keep-completed, get completed domjobinfo, and next query still can get statistics.
2. use --completed , get completed domjobinfo and clear statistics, later query will get none.

**Comment by yanqzhan** [comment]
]# avocado run --vt-type libvirt virsh.domjobinfo..keep_complete_test
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : a78289ac0cc95edc04dcd987744b3f9f00315ca4
JOB LOG    : /root/avocado/job-results/job-2021-11-10T05.44-a78289a/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.domjobinfo.normal_test.dump_action.keep_complete_test.running_state.vm_name: PASS (36.50 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 37.17 s

File: libvirt/tests/src/virsh_cmd/domain/virsh_domjobinfo.py [hunk 1]
```diff
@@ -183,7 +184,15 @@ def cmp_jobinfo(result, info_list, job_type, actions):
             except OSError:
                 pass
 
-    # Get completed domjobinfo
+    # Get completed domjobinfo with --keep-completed option, next completed domjobinfo gathering will still get statistics.
+    if keep_complete:
+        time.sleep(5)
+        vm_ref_tmp = "%s --completed --keep-completed" % vm_ref
+        ret_cmplt_tmp = virsh.domjobinfo(vm_ref_tmp, ignore_status=True, debug=True)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domjobinfo.py:191]: if ignore_status=False, virsh.domjobinfo can do assert itself, so line 192 and 193 are not necessary

Comment (yanqzhan) [reply libvirt/tests/src/virsh_cmd/domain/virsh_domjobinfo.py]: Yes, Thank you! Modified, please review again.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**